lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -0,0 +1,138 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function (intrinsic) {
+    var platform = intrinsic.JsBuiltIn;
+
+    let FunctionsEnum = {
+        ArrayValues: { className: "Array", methodName: "values", argumentsCount: 0, forceInline: true /*optional*/, alias: "Symbol.iterator"},
+        ArrayKeys: { className: "Array", methodName: "keys", argumentsCount: 0, forceInline: true /*optional*/ },
+        ArrayEntries: { className: "Array", methodName: "entries", argumentsCount: 0, forceInline: true /*optional*/ }
+    };
+ 
+    var setPrototype = platform.builtInSetPrototype;
+    var _objectDefineProperty = platform.builtInJavascriptObjectEntryDefineProperty;
+
+    // Object's getter and setter can get overriden on the prototype, in that case while setting the value attributes, we will end up with TypeError
+    // So, we need to set the prototype of attributes to null
+    var ObjectDefineProperty = function (obj, prop, attributes) {
+        _objectDefineProperty(obj, prop, setPrototype(attributes, null));
+    };
+    var CreateObject = platform.builtInJavascriptObjectCreate;
```

Comment:
Is there a way that we can simplify these aliases for all JsBuiltIns going forward? Like, perhaps we could make the names shorter or have a global standard library written in Javascript that all of these implementation files could use? Something along the lines of https://github.com/Microsoft/ChakraCore/pull/4083/files#diff-fa23cc60499db424efee4a57ac9bafddR105 would be my hope. Having everyone use _.keys or _.defineProperty rather than each files own (hopefully not but possibly) unique alias seems like it would make writing these JsBuiltIns much easier, especially for newer contributors.